### PR TITLE
Image slider

### PR DESCRIPTION
**Pull request by IvanaaVasic**
Main image slider, made with swiper library. And previous issues from comments fixed

### CODE REVIEW

File: src/components/ImagesSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          disableOnInteraction: false,
+        }}
+        pagination={{
+          clickable: true,
+        }}
+        navigation={true}
+        modules={[Autoplay, Pagination]}
+        className={styles.imgSwiper}
+      >
+        <SwiperSlide>
```
Comment (lukalazovic) [inline src/components/ImagesSlider.tsx]: You could use some API to fetch those images, also you can map SwiperSlide component in array and display data like that

File: src/components/NavBar.tsx [hunk 2]
```diff
@@ -1,27 +1,23 @@
 import React from "react";
 import Cart from "./Cart";
-import NavBarStyle from "../styles/NavBar.module.scss";
+import styles from "../styles/NavBar.module.scss";
 import Image from "next/image";
 import NextLink from "next/link";
+import data from "../../public/json/navData.json";
```
Comment (lukalazovic) [inline src/components/NavBar.tsx]: There is no need for making `navData.json` , you can use it like this

`const data =[
    { "name": "HOME", "path": "/" },
    { "name": "CHECKOUT", "path": "/checkout" },
    { "name": "CONTACT", "path": "/contact" },
    { "name": "PRODUCT", "path": "/product" }
  ]`

Comment (lukalazovic) [reply src/components/NavBar.tsx]: Also lets imagine that this data will come from API, can you make some IData interface?

**Review by lukalazovic — CHANGES_REQUESTED** [review]

File: src/components/ImagesSlider.tsx (hunk 1 above)
Comment (IvanaaVasic) [reply src/components/ImagesSlider.tsx]: This will be done with sanity, as you told me.

File: src/components/NavBar.tsx (hunk 2 above)
Comment (IvanaaVasic) [reply src/components/NavBar.tsx]: It's done.

File: src/components/ImagesSlider.tsx (hunk 1 above)
Comment (IvanaaVasic) [reply src/components/ImagesSlider.tsx]: I made array with images URLs and map SwiperSlide component.

**Review by lukalazovic — APPROVED** [review]

**Pull request merged**